jenkins/Jenkinsfile

```diff
@@ -108,6 +108,7 @@ pipeline {
                     environment {
                         CANDLEPIN_DATABASE = 'mysql'
                         CP_TEST_ARGS = '-r'
+                        OS_IMAGE = 'cs8'
```

Comment:
why is this only set at the standalone-mysql? should it also not be on the hosted-mysql stage?

Reply:
I want to just show how you can specify that OS, so i just set it to one job. So next person can see you could select OS or not and use default.

Reply:
If it is not functional, we should remove it. Anyone looking to do something like this will be able to tell by checking `docker/test --help`

Reply:
it has functionality, but same as default. Ok, I remove it.